vcl/main.vcl

```diff
@@ -12,6 +12,14 @@ sub vcl_recv {
 		set req.http.country-code = geoip.country_code3;
 	}
 
+	if (!req.http.ft-allocation-id && req.http.Cookie ~ "(^|; *)FTAllocation=([^;]+).*$") {
```

Comment:
https://github.com/Financial-Times/ft.com-cdn/blob/b82ca81befbb9b45d98534f09ac9da882cf2113b/src/vcl/cookie-to-header.vcl#L16-L18